oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/banyandb/client/request/TraceFetchRequest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.banyandb.client.request;
+
+import lombok.Getter;
+import lombok.Singular;
+import lombok.experimental.SuperBuilder;
+
+import java.util.List;
+
+@SuperBuilder
+@Getter
+public class TraceFetchRequest extends HasMetadata {
```

Comment:
Please break the unnecessary inheritance. Metadata should be client's constructor. In the future, metadata is used for database or tenancy isolation, you should not let a client shared.

Reply:
Metadata moved to client